drivers/volume/volume.go

```diff
@@ -751,3 +751,81 @@ func IsCSIDriverWithoutSnapshotSupport(pv *v1.PersistentVolume) bool {
 	}
 	return true
 }
+
+// GetNodeRegion - get the node region
+func GetNodeRegion() (string, error) {
+	nodes, err := core.Instance().GetNodes()
+	if err != nil {
+		logrus.Errorf("getNodeZones: getting node details failed: %v", err)
+		return "", fmt.Errorf("failed in getting the nodes: %v", err)
+	}
+	for _, node := range nodes.Items {
+		return node.Labels[v1.LabelTopologyRegion], nil
+	}
+	return "", fmt.Errorf("failed in getting the nodes: %v", err)
```

Comment:
When would this error msg be triggered as LabelTopologyRegion is always present for a node right?

Reply:
To avoid the compilation error, I added it.